tests/s3/test_object_workflow_operations.py

```diff
@@ -107,6 +111,40 @@ def create_bucket_put_objects(self, bucket_name, object_count):
 
         return obj_list
 
+    def create_bucket_put_get_object(self, bucket_name, obj_key_name, file_path, mb_count):
+        """
+        Function creates a bucket and uploads an object
+        and then retrieves object from specified S3 bucket.
+
+        :param bucket_name: Name of bucket to be created.
+        :param obj_key_name: Key name of an object.
+        :param file_path: Path of the file to be created and uploaded to bucket.
+        :param mb_count: Size of file in MBs.
+        """
+        if isinstance(obj_key_name, list):
```

Comment:
You are doing almost same operation in if and else. You can remove the if...else part and always send `obj_key_name` as list. Send it as list even if, it is 1 object.

Reply:
Passing object key names as list everytime .